mottak-arkiv-service/app/domain/arkivuttrekk_service.py

```diff
@@ -152,6 +155,44 @@ async def get_arkivkopi_status(arkivuttrekk_id: int, db: Session, is_object: boo
     return arkivkopi
 
 
+async def _get_overforingspakke_dto(invitasjon_id: int, db: Session) -> Optional[Overforingspakke_DTO]:
+    overforingspakke_dbo = overforingspakke_repository.get_by_invitasjon_id(db, invitasjon_id)
+    if not overforingspakke_dbo:
+        logger.info(f"No overforingspakke associated with invitasjon_id={invitasjon_id}")
+        return None
+    else:
+        return Overforingspakke_DTO(id=overforingspakke_dbo.id,
+                                    status=overforingspakke_dbo.status,
+                                    endret=overforingspakke_dbo.endret,
+                                    opprettet=overforingspakke_dbo.opprettet)
+
+
+async def _add_overforingspakke_to_invitasjon(invitasjon_dbo: Invitasjon_DBO, db: Session) -> Invitasjon_DTO:
+    overforingspakke_dto = await _get_overforingspakke_dto(invitasjon_dbo.id, db)
+    return Invitasjon_DTO(id=invitasjon_dbo.id,
+                          ekstern_id=invitasjon_dbo.ekstern_id,
+                          arkivuttrekk_id=invitasjon_dbo.arkivuttrekk_id,
+                          avgiver_epost=invitasjon_dbo.avgiver_epost,
+                          status=invitasjon_dbo.status,
+                          opprettet=invitasjon_dbo.opprettet,
+                          overforingspakke=overforingspakke_dto
+                          )
+
+
+async def _attach_overforingspakker_to_invitasjoner(invitasjoner: List[Invitasjon_DBO], db: Session) -> List[
+    Invitasjon_DTO]:
```

Comment:
Min editor forslår det slik (PEP8 E125)
```suggestion
async def _attach_overforingspakker_to_invitasjoner(invitasjoner: List[Invitasjon_DBO], db: Session) -> List[
        Invitasjon_DTO]:
```

Reply:
Done

Reply:
Trenger ikke bruke tid på dette nå, kan se på det senere når vi diskuterer kode standarder på ett senere tidspunkt.